db/DB_Insert.sql

```diff
@@ -1,168 +1,170 @@
 use teamC_Josh;
 
-INSERT INTO Capability(Capability_Name) VALUES ("Artificial Intelligence");
-INSERT INTO Capability(Capability_Name) VALUES ("Buisness Development & Marketing");
-INSERT INTO Capability(Capability_Name) VALUES ("Product");
-INSERT INTO Capability(Capability_Name) VALUES ("Cyber Security");
-INSERT INTO Capability(Capability_Name) VALUES ("Data and Analytics");
-INSERT INTO Capability(Capability_Name) VALUES ("Engineering");
-INSERT INTO Capability(Capability_Name) VALUES ("Experienced Design");
-INSERT INTO Capability(Capability_Name) VALUES ("Delivery Management");
-INSERT INTO Capability(Capability_Name) VALUES ("Platforms");
-
-INSERT INTO Band(Band_Name,Band_Level, Band_Training, Band_Competencies, Band_Responsibilites) 
+INSERT INTO capability(capabilityName) VALUES ("Artificial Intelligence");
+INSERT INTO capability(capabilityName) VALUES ("Business Development & Marketing");
```

Comment:
You can insert multiple rows with a single statement, like `INSERT INTO [Table] VALUES ([values...]), ([values...]), ...;`